package.json

```diff
@@ -12,10 +12,10 @@
 		"prestart": "run-p setup clean db:dev",
 		"clean": "rimraf coverage build tmp src/**/*.js src/**/*.js.map test/**/*.js test/**/*.js.map",
 		"init": "run-p db",
-		"start": "run-p start:main start:worker",
+		"start": "run-p spa:dev start:main start:worker",
 		"start:main": "tsnd -r tsconfig-paths/register --watch=.env*,db/config.json --inspect=0.0.0.0:9229 --respawn --transpile-only -- src/main.ts",
 		"start:worker": "tsnd -r tsconfig-paths/register --watch=.env*,db/config.json --inspect=0.0.0.0:9230 --respawn --transpile-only -- src/worker.ts",
-		"start:production": "run-p start:main:production start:worker:production",
+		"start:production": "run-p spa:production start:main:production start:worker:production",
```

Comment:
`spa:production` should not be here, as in production, we don't need it it. We should copy the compiled assets during build time not run time.

Reply:
`spa:production` removed and added this step in `build`